gatilegrid/tilegrids.py

```diff
@@ -312,6 +312,16 @@ def getClosestZoom(self, resolution, unit='meters'):
                 hi = mid
             else:
                 lo = mid + 1
+        return lo, hi
+
+    def getClosestZoom(self, resolution, unit='meters'):
+        """
+        Return the closest zoom level for a given resolution
+        Parameters:
+            resolution -- max. resolution
+            unit -- unit for output (default='meters')
+        """
+        lo, hi = self.getZoomLevelRange(resolution, unit)
```

Comment:
Please rename `getZoomLevelRange` to `_getZoomLevelRange` to denote that it should be a private method and should not be used directly.